docs/docusaurus/siteConfig.js

```diff
@@ -20,49 +20,55 @@ const users = [
     caption: 'User1',
     // You will need to prepend the image path with your baseUrl
     // if it is not '/', like: '/test-site/img/docusaurus.svg'.
-    image: '/img/docusaurus.svg',
-    infoLink: 'https://www.facebook.com',
+    image: '/img/magma-logo.svg',
+    infoLink: 'https://magmacore.org',
     pinned: true,
   },
 ];
 
 const siteConfig = {
-  title: 'Magma', // Title for your website.
+  title: 'Magma Documentation', // Title for your website.
+  disableTitleTagline: true, 
   tagline: 'Bring more people online by enabling operators with open, flexible, and extensible network solutions',
-  url: 'https://magma.github.io', // Your website URL
-  baseUrl: '/magma/', // Base URL for your project */
-  // For github.io type URLs, you would set the url and baseUrl like:
-  //   url: 'https://facebook.github.io',
-  //   baseUrl: '/test-site/',
 
   // Used for publishing and more
   projectName: 'magma',
-  organizationName: 'magma',
+  organizationName: 'magma',  
+  url: 'https://magmacore.org', // Your website URL
+  baseUrl: '/', // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
   // For top-level user or org sites, the organization is still the same.
   // e.g., for the https://JoelMarcey.github.io site, it would be set like...
   //   organizationName: 'Facebook'
 
-  docsUrl: 'docs',
+  //docsUrl: 'docs',
```

Comment:
do we not need this anymore?

Reply:
We won't need that for the Netlify instance b/c we're pointing directly to the docs directory.